CONTRIBUTING.md

```diff
@@ -278,6 +278,19 @@ Below is a checklist of things to be mindful of when implementing a new instrume
 - Testing
   - When adding a new instrumentation remember to update `tox.ini` adding appropriate rules in `envlist`, `command_pre` and `commands` sections
 
+### Update supported instrumentation package versions
+
+- Update the respective instrumentation `pyproject.toml` file in _instruments_ entry under `[project.optional-dependencies]` section
```

Comment:
maybe file the first two under a bullet for "In the project directory" and the last I think needs to be "In the repository root" right?

Reply:
Yes, first two are inside instrumentation package directory.